libraries/botbuilder-azure-blobs/src/blobsTranscriptStore.ts

```diff
@@ -0,0 +1,230 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import getStream from 'get-stream';
+import pmap from 'p-map';
+import { Activity, PagedResult, TranscriptInfo, TranscriptStore } from 'botbuilder';
+import { maybeCast } from './maybeCast';
+import { sanitizeBlobKey } from './sanitizeBlobKey';
+
+import {
+    ContainerClient,
+    ContainerListBlobHierarchySegmentResponse,
+    StoragePipelineOptions,
+} from '@azure/storage-blob';
```

Comment:
Order alphabetically?

Reply:
I believe these azure deps are in alphabetical order 😄

Reply:
I was thinking of package name 😄 
```js
'@azure/storage-blob'
'botbuilder'
'get-stream'
'p-map'

// locals
'./maybeCast'
'./sanitizeBlobKey'
```